pages/feed/index.tsx

```diff
@@ -41,22 +44,35 @@ const Feeds = () => {
       return { repoName: repo, repoIcon: `https://www.github.com/${orgName}.png?size=300`, full_name };
     });
 
+
   useEffect(() => {
     if (selectedRepo) {
       router.push(`/feed?repo=${selectedRepo}`);
     }
+
     if (highlightId) {
-      router.push(`/feed/${id}`);
+      setOpenSingleHighlight(true);
```

Comment:
Hey @TAKANOME-DEV , you might need to add 
```javascript
router.push(`/feed/${id}`);
```
below the `setOpenSingleHighlight(true)` to persist the `{id}` on page reload.

Reply:
Yeah, but it causes errors in the console. The issue was caused by this:
```js
if (!highlightId && !selectedRepo) {
  router.push("/feed");
}
```